### ISSUE TITLE
RPi3: added widgets but I can't select them -> EstuaryMod v2

### ISSUE THREAD

**Issue by walterheisenberg** [BigNoid/script.skinshortcuts#235]
I don't know, if this is a bug in skinshortcuts or in EstuaryMod v2:
If I add widgets, I can't select them. The author of the skin, guilouz said that it is no skin issue, because it is only happening on RPi.

more info and forum post:
https://forum.kodi.tv/showthread.php?tid=178294&pid=2657452#pid2657452
Thx for making this clear

Update: Angelinas found the issue: https://forum.kodi.tv/showthread.php?tid=306757&pid=2677994#pid2677994
but no fix yet

**Comment by stevenleadbeater** [comment]
I've got the same issue. According to the addons repo, it is using version 0.6.5 of the skinshortcuts script. https://github.com/Guilouz/repository.guilouz/blob/master/skin.estuary.modv2/addon.xml for me this is specifically with version 18 of kodi, so it's an alpha to be clear (estuary mod v2 is kodi 18 only). I'm going to try upgrading the skinshortcuts version to current and seeing if that helps. I have tried this with various millhouse builds for libreelec. Failing that, does anyone have any idea if this issue is related to skinshortcuts or is it upstream in  kodi itself?

**Comment by stevenleadbeater** [comment]
So after a bit of mucking around, you can upgrade to version 18 from marceveldts beta repository and break your main menu if you want. You have to go back to an old version then and upgrade to the one in the main kodi repo... was a pest to fix, dont do it!

**Comment by walterheisenberg** [comment]
updated first post: issue found, no fix yet

**Comment by KrX3D** [comment]
i have the same problem, any fix yet?

**Comment by Happyarms** [comment]
A fix would be nice, but i would also take any workarounds.

**Comment by TehTux** [comment]
> updated first post: issue found, no fix yet

I don't think that is the problem. I found these properties on macOS too. There it works...

Workaround:
Disable the option "Share menu between compatible skins" in "Skin Shortcuts" settings

**Comment by piejanssens** [comment]
Same problem with Vero4K (OSMC), no issue on macOS.
So it's not something specific to RPi...
@BigNoid @mvalec @marcelveldt 
Some attention to this long standing critical issue would be great.
Tell us what you need, logs, generated script-skinshortcuts-includes.xml, ...?

In my case I cannot select/navigate the second row of widgets of my custom menu items.
